pom.xml

```diff
@@ -290,6 +290,13 @@
               <shadedArtifactAttached>true</shadedArtifactAttached>
               <shadedClassifierName>shaded</shadedClassifierName>
               <relocations>
+                <relocation>
+                  <pattern>com.redhat.insights</pattern>
+                  <shadedPattern>${shade.prefix}</shadedPattern>
```

Comment:
Should we shade the client with the pattern `${shade.prefix}.com.redhat.insights`? For example, the com.redhat.insights.reports package is being added to the shaded jar under com/redhat/insights/agent/shaded/reports/, where other dependencies have the full package hierarchy. This is unlikely to cause conflicts, but I want to check whether this was an intentional choice.

Reply:
This was an intentional choice just to prevent horrendously long package names - but I'm happy to take your suggestion if we feel that uniformity is more important

Reply:
It's fine, I doubt we'll encounter any dependencies that have package names that are just simply `reports` for example.